### DISCUSSION TITLE
Is webpack caching safe to use for production builds ?

### DISCUSSION THREAD

**Discussion by zakuru** [webpack/webpack#16723] [Q&A]
Hi,

I saw this section related to caching setup in CI/CD - but is is safe to use for production builds ? 
![image](https://user-images.githubusercontent.com/171174/218663879-1bf95743-4ac9-4d2e-bb28-e375ef2125cc.png)

https://webpack.js.org/configuration/cache/

Thanks

**Comment by alexander-akait** [comment]
And yes and no, if you have sensitivity infromation (tokens/passwords/etc) in production files, it will be unsafe, if you don't have any sensitivity infromation it will be safe

**Reply by zakuru** [reply]
No our client side code wont hold any sensitive information such as the one you mentioned thank you @alexander-akait

**Reply by zakuru** [reply]
would it be worth motioning in the webpack cache documentation ? 
Now its a bit unclear if we can use it or not and under which condition as you described
Thanks a lot btw 
https://webpack.js.org/configuration/cache/

**Reply by alexander-akait** [reply]
If you think it is very import for you can send a PR, thank you
